src/go/k8s/Makefile

```diff
@@ -72,32 +77,39 @@ generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="../../../licenses/boilerplate.go.txt" paths="./..."
 
 # Build the docker image
-docker-build: test
-	docker build -f Dockerfile -t ${IMG} ../
-
-# Push the docker image
-docker-push:
-	docker push ${IMG}
+docker-build:
+	docker build -f Dockerfile -t ${OPERATOR_IMG_LATEST} ../
+
+docker-tag-release:
+	docker tag ${OPERATOR_IMG_LATEST} ${OPERATOR_IMG_RELEASE}
+	docker tag ${CONFIGURATOR_IMG_LATEST} ${CONFIGURATOR_IMG_RELEASE}
+
+# Push to dockerhub
+docker-push-release:
```

Comment:
I agree. The main issue is avoiding the token to be printed on the console. I'd like to port this Makefile to Task soon, to be consistent with automation for other projects (redpanda, rpk, coproc, dashboard, etc.). Once this port is done, this logic would be in the taskfile rather than the workflow